registry.json

```diff
@@ -12,6 +12,20 @@
         "telegram": "AdaStak_Official",                              
         "operatorName": "Will",		                                   
         "github": "wrp0"                                                      
+    },
+    {
+        "ticker": "Z3LDA",                                                      
+        "poolId": "263578cc21e4acdfb86ca47a90fbb9200a203e45975e6fea423fb438",
+        "social": {                                                            
+            "twitter": "Z3ldaP",
+            "website": "https://www.facebook.com/z3ldastakingpool",
+            "facebook": "https://www.facebook.com/z3ldastakingpool",  
+            "instagram": " ",
+            "youtube": " "
+        },
+        "telegram": "z3ldastakepooll",                              
```

Comment:
This could not be found. It should be a users telegram handle - not the name of a group / page.